std_msgs/Header header
openpose_ros/Person[] people

